### PR TITLE
chore(cs) Update to latest PHPCSFixer API

### PR DESCRIPTION
**Pull request by Hywan**

### CODE REVIEW

**Comment by Grummfy** [comment]
perhaps we can also add a [.gitattributes](https://git-scm.com/docs/gitattributes) or a a [.editorconfig](http://editorconfig.org/) ?

File: .php_cs [hunk 1]
```diff
@@ -1,31 +1,20 @@
...
+            'concat_space'                       => ['spacing' => 'one'],
+            'native_function_casing'             => true,
+            'no_blank_lines_after_class_opening' => true,
+            'no_unused_imports'                  => true,
+            'no_unused_imports'                  => true,
+            'no_whitespace_in_blank_line'        => true,
+            'ordered_imports'                    => true,
+            'phpdoc_no_access'                   => true,
+            'self_accessor'                      => true
+        ]);
```
Comment (jubianchi) [inline .php_cs]: missing newline.

File: .php_cs [hunk 2]
```diff
@@ -1,31 +1,20 @@
...
-            'phpdoc_no_access',
-            'remove_leading_slash_use',
-            'remove_leading_slash_uses',
-            'self_accessor',
-            'short_array_syntax',
-            'spaces_cast',
-            'unused_use',
-            'whitespacy_lines'
-        ])
-        ->finder(
```
Comment (jubianchi) [inline .php_cs]: You have to define the finder in the new configuration

Comment (Hywan) [reply .php_cs]: The `finder` method no longer exists.

Comment (jubianchi) [reply .php_cs]: see here https://github.com/atoum/telemetry-extension/blob/master/.php_cs ;)

**Pull request merged**